maths/prime_numbers.py

```diff
@@ -13,16 +13,16 @@ def primes(max: int) -> List[int]:
     >>> primes(1_000_000)[-1]
```

Comment:
Please change this to __list(primes(1_000_000))[-1]__ so this doctest passes.

Please add doctests for __list(primes(0))__, __list(primes(-1))__, __list(primes(-10))__

You can try these on your own computer with __python3 -m doctest -v maths/prime_numbers.py__